.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,70 @@
+name: Trivy Security Scanner
+on:
+  push:
+    branches:
+      - 6-22.04
+  pull_request:
+jobs:
+  build:
+    name: Build rock
+    uses: canonical/data-platform-workflows/.github/workflows/build_rock.yaml@v29.0.1
+  scan:
+    name: Trivy scan and SBOM Generation
+    needs: build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repo
```

Comment:
Since we don't use some extra permissions, maybe we can enforce some more secure permissions like `read` as recommended [here](https://github.com/actions/checkout?tab=readme-ov-file#recommended-permissions)?